preppal-be/routes/authApi.ts

```diff
@@ -0,0 +1,52 @@
+const expressAuthApi = require("express");
+const routerAuthApi = expressAuthApi.Router();
+const jwtAuthApi = require("jsonwebtoken");
+const configAuthApi = require("../configs/secrets.ts")
+
+const UserAuth = require('../models/user.ts');
+
+/**
+ * GET - Get all accounts
+ */
+routerAuthApi.get("/", async (req, res) => {
+    try {
+        const users = await UserAuth.find().select("-password");
+        res.json(users);
+    } catch (error) {
+        console.error(error.message);
+        res.status(500).send("Server error");
+    }
+});
+
+/**
+ * POST - Login to an account
+ */
+routerAuthApi.post("/", async(req, res) => {
+    try {
+        const { username, password } = req.body;
+        let user = await UserAuth.findOne({ username: username });
+        if (!user || password != user.password) {
+            return res.status(400).json({ errors: [{ msg: "Invalid username or password!" }] });
+        }
+
+        const payload = {
+            user: {
+                id: user.id,
+            },
+        };
+
+        jwtAuthApi.sign(payload, configAuthApi.jwtSecret, { expiresIn: 3600 * 24 },
+            (err, token) => {
+                if (err) throw err;
+                res.json({ token });
+            }
+        );
+        res.status(200).send("logged in");
```

Comment:
Remove this line. 

I was using this line to test API calls.
Tried to login with through the front end and found that this line was causing it to break.

Reply:
done